flowkit_jwt_generator/flowkit_jwt_generator/jwt.py

```diff
@@ -210,11 +210,12 @@ def squashed_scopes(scopes: List[str]) -> Iterable[str]:
 def generate_token(
     *,
     flowapi_identifier: Optional[str] = None,
+    flowauth_identifier: Optional[str] = None,
     username: str,
     private_key: Union[str, _RSAPrivateKey],
     lifetime: datetime.timedelta,
     claims: List[str],
-    compress: bool = True,
+    compress: bool = False,
```

Comment:
Why don't we compress anymore?

Reply:
Ah, I can answer that one. We started compressing the token claims when the old permissions got so big that tokens no longer fitted in http headers. It wasn't ideal (meant that users could no longer directly check the permissions granted by a token just from the decoded token - they would also have to decompress the claims), but was a necessary workaround at the time. But with the new (smaller) permissions, that workaround is no longer necessary.